### ISSUE TITLE
Nullreference exception on export-key

### ISSUE THREAD

**Issue by TFTomSun** [AppCoreNet/SigningTool#5]
```
dotnet-signtool sn export-key keyfile.snk public.snk
```

```
 
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at AppCore.SigningTool.Commands.Sn.SnExportPublicKeyCommand.ParseAssemblyHashAlgorithm(String value) in /home/vsts/work/1/s/src/AppCore.SigningTool/Commands/Sn/SnExportPublicKeyCommand.cs:line 58
   at AppCore.SigningTool.Commands.Sn.SnExportPublicKeyCommand.<Configure>b__10_0(ValidationContext vc) in /home/vsts/work/1/s/src/AppCore.SigningTool/Commands/Sn/SnExportPublicKeyCommand.cs:line 42
   at McMaster.Extensions.CommandLineUtils.Validation.DelegateValidator.McMaster.Extensions.CommandLineUtils.Validation.IOptionValidator.GetValidationResult(CommandOption option, ValidationContext context)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.GetValidationResult()
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.ExecuteAsync(String[] args, CancellationToken cancellationToken)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at AppCore.SigningTool.Program.Main(String[] args) in /home/vsts/work/1/s/src/AppCore.SigningTool/Program.cs:line 36
```

**Comment by prochnowc** [comment]
Can you please try to specify the default signing algorithm explicitly? I have some fixes pending anyways. I will get into this on monday.

**Comment by TFTomSun** [comment]
that workaround solved the issue

**Comment by prochnowc** [comment]
Fixed by #7 . Please update to v0.3.0.
